src/main/java/com/eppo/sdk/EppoClient.java

```diff
@@ -497,22 +502,52 @@ public Exception logNonBanditAction(
 
             String variationValue = assignmentResult.getVariation().getTypedValue().toString();
 
+            Map<String, Double> subjectNumericAttributes = numericAttributes(subjectAttributes);
+            Map<String, String> subjectCategoricalAttributes = categoricalAttributes(subjectAttributes);
+            Map<String, Double> actionNumericAttributes = numericAttributes(actionAttributes);
+            Map<String, String> actionCategoricalAttributes = categoricalAttributes(actionAttributes);
+
             this.eppoClientConfig.getBanditLogger().logBanditAction(new BanditLogData(
               assignmentResult.getExperimentKey(),
               variationValue,
               subjectKey,
-              subjectAttributes,
               actionString,
-              actionAttributes,
               null,
-              null
+              null,
```

Comment:
```
  public String action;
  public Float actionProbability; --> null
  public String modelVersion; --> null
```

What's an example of a "NonBandit" action string?

Reply:
Any action not selected by the bandit, such as something selected by a recommendation algorithm.